package.json

```diff
@@ -12,5 +12,33 @@
     "adapt-core"
   ],
   "license": "GPL-3.0",
-  "bugs": "https://github.com/adaptlearning/adapt-contrib-core/issues"
+  "bugs": "https://github.com/adaptlearning/adapt-contrib-core/issues",
+  "private": true,
+  "devDependencies": {
+    "@semantic-release/git": "^10.0.1",
+    "conventional-changelog-eslint": "^3.0.9",
+    "semantic-release": "^19.0.3"
+  },
+  "release": {
+    "plugins": [
+      "@semantic-release/release-notes-generator",
+      "@semantic-release/npm",
```

Comment:
We don't publish these to npm yet.

Reply:
https://semantic-release.gitbook.io/semantic-release/extending/plugins-list

Reply:
Hi Oli, documentation had told me that the `"private": true` package.json setting bypasses the npm publish. But I did think it still had to be included. I'll test if it works without that package